webhook-system/README.md

```
@@ -0,0 +1,318 @@
+  "maxRetries": 5,
+  "timeout": 10000,
+  "headers": {
+    "Authorization": "Bearer token-xyz",
+    "X-Custom-Header": "value"
+  }
+}
+```
+
+## Delivery Flow
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)